terminology/README.md

```diff
@@ -0,0 +1,15 @@
+# Terminology
+
+The language we use has power. Being as concise and descriptive as possible helps to lessen ambiguity. This leads to shared understanding and better outcomes.
+
+## Terms
+
+* Use `blocklist`, and not `blacklist`.
+* Use `allowlist`, and not `whitelist`.
+* Avoid using `master` and `slave`. 
+	* For GitHub, use a term such as `main`, or `publish`, `production`, or `trunk`.
```

Comment:
Some projects use other services for their git remotes, but this applies to git in general.

```suggestion
	* For git, use a term such as `main`, or `publish`, `production`, or `trunk`.
```

Reply:
Oops, I see this comment was already made. Don't mind me.